### DISCUSSION TITLE
Welcome to Scribble Discussions!

### DISCUSSION THREAD

**Discussion by awcodes** [awcodes/scribble#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by monteduro** [comment]
Hi Adam, I've been tinkering with the TipTap editor for a while, and only now have I discovered this package (which has the slash command I've been wanting so much). But is this a parallel project, or will it replace the TipTap integration of the other package?

**Reply by awcodes** [reply]
Eventually, this will replace it. But there will be some crossover support for both until that happens.
